### PR TITLE
Changed the pipelines limitation note to mention just one issue (GCR)

### PR DESCRIPTION
**Pull request by sarahmaddox**
Fixes https://github.com/kubeflow/website/issues/1247

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1434)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
/assign @gaoning777 

@gaoning777 Please would you review this PR and check that it correctly describes the implications of the pipelines issue.

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1434--competent-brattain-de2d6d.netlify.com/docs/pipelines/pipelines-quickstart/#deploy-kubeflow-and-open-the-pipelines-ui

File: layouts/shortcodes/pipelines-compatibility.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <p><i>Due to
-<a href="https://github.com/kubeflow/pipelines/issues/1700">kubeflow/pipelines#1700</a>
-and
-<a href="https://github.com/kubeflow/pipelines/issues/337">kubeflow/pipelines#337</a>,
```
Comment (gaoning777) [inline layouts/shortcodes/pipelines-compatibility.html]: The KFP SDK currently prepares the credentials for GCP only. So, yes, the container builder only supports GCR. However, it does not mean the containers have to be in GCR once built. As long as the credentials are set up correctly to fetch the image, other registries also work.

Comment (sarahmaddox) [reply layouts/shortcodes/pipelines-compatibility.html]: Thanks @gaoning777 I've adjusted the wording. PTAL.

**Pull request merged**